src/bundles/ipfs-provider.js

```diff
@@ -204,42 +216,83 @@ const bundle = {
   /**
    * @param {State} state
    */
-  selectIpfsInitFailed: state => state.ipfs.failed,
+  selectIpfsInitFailed: state => state.ipfs.failed
+}
 
-  doInitIpfs: () => async (store) => {
-    await initIPFS(store)
-  },
+/**
+ * @typedef {import('redux-bundler').Actions<typeof actions>} Actions
+ * @typedef {import('redux-bundler').Context<State, Message, {}, Extra>} Context
+ */
 
-  doStopIpfs: () => async (store) => {
+const actions = {
+  /**
+   * @returns {function(Context):Promise<void>}
+   */
+  doInitIpfs: () => async (context) => {
```

Comment:
Turns out what redux-bundle passes into `doX` style things isn't a store but rather it's own context thing that contains `store` (see https://reduxbundler.com/api-reference/bundle#bundle-doxA) I think I originally was calling it a store as well, but then it got confusing because in some cases it is store and in others it isn't.